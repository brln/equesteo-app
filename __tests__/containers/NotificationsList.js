import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import NotificationsListContainer from '../../containers/NotificationsList'


const mockStore = configureStore([thunk])

test('render', () => {
  const store = mockStore(fromJS({
    pouchRecords: {
      notifications: {},
    },
    localState: {},
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <NotificationsListContainer />
    </Provider>).toJSON()
  expect(tree).toMatchSnapshot()
})

