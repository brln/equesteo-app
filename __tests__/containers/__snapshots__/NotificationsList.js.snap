// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text>
      All caught up!
    </Text>
  </View>
</View>
`;
